agent/replacecall/src/main/java/randoop/instrument/ReplaceCallAgent.java

```diff
@@ -68,12 +70,30 @@ public class ReplaceCallAgent {
   /** The file from which to read the user replacements for replacing calls. */
   @SuppressWarnings("WeakerAccess")
   @Option("file listing methods whose calls to replace by substitute methods")
-  public static @Nullable Path replacement_file = null;
+  public static @MonotonicNonNull Path replacement_file = null;
 
   /** Exclude transformation of classes in the listed packages. */
   @SuppressWarnings("WeakerAccess")
   @Option("file listing packages whose classes should not be transformed")
-  public static @Nullable Path dont_transform = null;
+  public static @MonotonicNonNull Path dont_transform = null;
+
+  /**
+   * Default replacement methods that should not be used. A comma-separated list of method
+   * signatures in colon/JVM-descriptor form, e.g. "java.lang.System.exit:(I)V". JVM method
+   * descriptors are defined in the Java Virtual Machine Specification, section 4.3.3.
+   */
+  @SuppressWarnings("WeakerAccess")
+  @Option("methods not to replace")
+  public static @MonotonicNonNull String replacecall_exclude = null;
+
+  /**
+   * File listing method to exclude from default replacements. Each line is a method signature in
+   * colon/JVM-descriptor form, e.g. "java.lang.System.exit:(I)V". JVM method descriptors are
+   * defined in the Java Virtual Machine Specification, section 4.3.3.
+   */
+  @SuppressWarnings("WeakerAccess")
+  @Option("file listing methods not to replace, one per line")
+  public static @MonotonicNonNull Path replacecall_exclude_file = null;
```

Comment:
There is already a `--dont-transform` command-line argument that excludes classes from replacement.  Why is a new command-line argument required?  How about permitting that file to contain both packages and methods?  That is consistent with how replacements are defined, and it avoids the need for a new command-line option.